database/migrations/2017_10_24_085111_create_order_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
I think this file is out of the "edit product" scope. It is better to remove it for now :)